src/cnxml/deserialize.js

```diff
@@ -177,13 +177,14 @@ function emphasis(el) {
 function xref(el) {
     const target = el.getAttribute('target-id') || null
     const url = el.getAttribute('url') || null
+    const cmlnleCase = el.getAttribute('cmlnle-case') || null
```

Comment:
This won't work, you need to use [`getAttributeNS`](https://developer.mozilla.org/en-US/docs/Web/API/Element/getAttributeNS),

Reply:
There was type `-` instead of `:`
`getAttributeNS('cmlnle', 'case)` was not working but `getAttribute('cmlnle:case')` is ok.

Reply:
`getAttribute('cmlnle:case')` also doesn't work:

```js
const document = `<?xml version="1.0" encoding="utf-8"?>
<document xmlns="http://cnx.rice.edu/cnxml" xmlns:cmlnle="http://katalysteducation.org/cmlnle/1.0" xmlns:n1="http://katalysteducation.org/cmlnle/1.0">
    <test cmlnle:case="nominative" />
    <test n1:case="accusative" />
</document>`

const tree = new DOMParser().parseFromString(d, 'application/xml')

console.log(t.children[0].children[0].getAttribute('cmlnle:case'))
// => nominative
console.log(t.children[0].children[1].getAttribute('cmlnle:case'))
// => null
```

The reason that `getAttributeNS` didn't work for you is because you used it wrong. `cmlnle` is _not_ a namespace name, its a namespace _prefix_, which the user is free to choose (as I did here, using both `cmlnle` and `n1` to refer to the same namespace). The namespace is actually `http://katalysteducation.org/cmlnle/1.0`. Continuing with example:

```js
console.log(t.children[0].children[0].getAttributeNS('http://katalysteducation.org/cmlnle/1.0', 'case'))
// => nominative
console.log(t.children[0].children[1].getAttributeNS('http://katalysteducation.org/cmlnle/1.0', 'case'))
// => accusative
```

Reply:
Strange, I've tested on live editor and `getAttribute('cmlnle:case')` is working.

Reply:
Apparently it does when you use the same namespace prefix as is used in the document, but it won't work if you use a different one, as I've shown.